common/src/components/accordion/Section.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react";
+import { ChevronDownIcon, ChevronRightIcon } from "@heroicons/react/solid";
+
+export interface ExternalSectionProps {
+  title: string;
+  subtitle?: string;
+  Icon?: React.ComponentType<React.SVGProps<SVGSVGElement>>;
+}
+
+interface InternalSectionProps extends ExternalSectionProps {
+  open?: boolean;
```

Comment:
Since `isOpen` is managed by internal state as well, maybe it would be more clear to call this `defaultOpen` or `initialOpen` or something like that?

Reply:
will do